packages/mobile/src/sentry/Sentry.ts

```diff
@@ -16,11 +16,7 @@ export async function installSentry() {
     Logger.info(TAG, 'installSentry', 'Sentry URL not found, skiping instalation')
     return
   }
-  await Sentry.config(SENTRY_URL).install()
-  Sentry.setTagsContext({
-    environment: DeviceInfo.getBundleId(),
-    react: true,
-  })
+  Sentry.init({ dsn: SENTRY_URL, environment: await DeviceInfo.getBundleId() })
```

Comment:
New sentry version changed the API a bit

Reply:
`DeviceInfo.getBundleId()` is still synchronous:
https://github.com/react-native-community/react-native-device-info/blob/v4.0.1/src/index.ts#L273-L282